commit b268f77e859451750d2f75840f55a5283fb4edd1 (refs/changes/80/3180/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-01-09T13:41:56-08:00 (1 year, 9 months ago)
    
    joyent/node-zkstream#31 should try to resume sessions for a while even after expiry time elapses

diff --git a/lib/zk-session.js b/lib/zk-session.js
index a3489d5..d6e8941 100644
--- a/lib/zk-session.js
+++ b/lib/zk-session.js
@@ -80,7 +80,7 @@ ZKSession.prototype.resetExpiryTimer = function () {
 	this.zs_expiryTimer.ref = setTimeout(function () {
 		self.zs_expiryTimer.ref = undefined;
 		self.zs_expiryTimer.emit('timeout');
-	}, this.zs_timeout);
+	}, this.zs_timeout * 2);
 };
 
 ZKSession.prototype.getTimeout = function () {
diff --git a/package.json b/package.json
index 23daeef..664c023 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zkstream",
-  "version": "0.10.1",
+  "version": "0.10.2",
   "description": "",
   "repository": {
     "type": "git",
